llvm/lib/Transforms/Utils/LowerWGLocalMemoryCall.cpp

```diff
@@ -0,0 +1,114 @@
+//===-- LowerWGLocalMemoryCall.cpp - SYCL kernel local memory allocation --===//
```

Comment:
Please move this to llvm/lib/SYCLLowerIR where SYCL-specific lowering passes congregate.

Reply:
Done